src/main/java/com/ericsson/eiffel/remrem/semantics/eiffelSchemas/LocalRepo.java

```diff
@@ -0,0 +1,76 @@
+package com.ericsson.eiffel.remrem.semantics.eiffelSchemas;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+
+import org.apache.commons.io.FileUtils;
+
+/**
+ * This class is used to Iterate over the Eiffel schemas that are cloned from
+ * Eiffel repository
+ * 
+ * @author xdurvak
+ *
+ */
+public class LocalRepo {
+	private HashMap<String, File> jsonFiles;
+	private File localSchemasPath;
+	private static ArrayList<String> jsonFileNames;
+	
+	public LocalRepo(File localSchemasPath) {
+		this.localSchemasPath=localSchemasPath;
+	}
+
+	/**
+	 * This method is used to read Eiffel Schemas that are cloned from Eiffel
+	 * Repo
+	 */
+	
+	public void readSchemas() {
+		try {
+			FileUtils.cleanDirectory(new File(EiffelConstants.USER_DIR + "\\" + EiffelConstants.INPUT_EIFFEL_SCHEMAS));
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		jsonFileNames = new ArrayList<String>();
+		jsonFiles = new HashMap<String, File>();
+		String filePath = localSchemasPath + EiffelConstants.SCHEMA_LOCATION;
+		File file = new File(filePath);
+		if (file.isDirectory()) {
+			loadEiffelSchemas(filePath, "");
+		}
+		Iterator<String> iter = jsonFileNames.iterator();
```

Comment:
Remove this part from the readSchemas method. This is no longer "read" this is modify.